CHANGES.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 4.11 (unreleased)
 -----------------
 
+- Support messages with line separation `\r\n`
```

Comment:
I suggest to change the change entry to: "Use standard conforming \`\`\r\n\`\` line endings. This may require adaptations in tests".